src/components/overblikk.tsx

```diff
@@ -57,40 +72,42 @@ const Overblikk = () => {
     }, [oppfolgingsstatus?.veilederId]);
 
     // Midlertidig data i overblikk, må endres etterhvert som vi får bedre innsikt i hva som trengs i oversikten
-    const veilederNavn: StringOrNothing = veileder?.navn;
-    const telefon: StringOrNothing = person?.telefon?.[0]?.telefonNr; // Henter kun ut 1 tlfnr, flere kan være registrert
+    const veilederData: VeilederData | null | undefined = veileder;
+    const telefon: StringOrNothing = person?.telefon?.find((entry) => entry.prioritet === '1')?.telefonNr;
     const antallBarn: StringOrNothing = person?.barn?.length.toString();
-    const oppfolgingsenhet: OppfolgingEnhet | undefined = oppfolgingsstatus?.oppfolgingsenhet;
-    const talespraak: StringOrNothing = tolk?.talespraak;
+    const taletolk: OrNothing<TilrettelagtKommunikasjonData> = tolk;
     const sivilstatus: StringOrNothing = person?.sivilstandliste?.[0]?.sivilstand;
-    const brukersMaal: StringOrNothing = oppfolgingsstatus?.hovedmaalkode;
-    const ytelserAktivitetsFase: StringOrNothing = ytelser?.vedtaksliste?.[0]?.aktivitetsfase; // Henter kun ut 1 ytelse, flere kan være registrert
-    const geografiskEnhet: OrNothing<Enhet> = person?.geografiskEnhet;
+    const brukersMaal: OrNothing<Hovedmal | ArenaHovedmalKode> = oppfolgingsstatus?.hovedmaalkode;
```

Comment:
Ikke så åpenbart kanskje, men brukers mål og hovedmål er ikke det samme 😁 
Hovedmål settes av veilederen og går på om bruker skal beholde arbeid eller skaffe arbeid, mens brukers mål settes gjerne av brukeren selv, [i aktivitetsplanen](https://navikt.github.io/aktivitetsplan/#/12345678910), og er mer et ønske for eksempel et spesifikt yrke eller "jeg vil ta truckførerlappen".
Så dette som kommer fra Arena er hovedmål, ikke brukers mål

Reply:
Ser dere endra tittelen i info-komponenten, så antar noen andre kanskje har forklart dette allerede 😅